app-frontend/config/webpack/environments/production.js

```diff
@@ -17,6 +17,20 @@ module.exports = function (_path) {
             filename: '[name].[chunkhash].js'
         },
         plugins: [
+        // Return error code when build fails.

```

Comment:
I'm leaning towards keeping things inline. I don't think webpack-fail-plugin will be getting many more fixes since the plugin itself has been deprecated. Additionally, having this single inline fix keeps all of the changes localized to one file. I think will be easier to delete a block of code than removing a dependency (and references to it) from multiple locations. That being said, if there are legibility concerns I can move the function definition elsewhere.